### PR TITLE
remove Juniper plugin references

### PR DESCRIPTION
**Pull request by DaanHoogland**
a bit hard to define which are just plugin references and which are architectural best practices.
this implements [cloudstack#7690](https://github.com/apache/cloudstack/issues/7690)

----
:books: Documentation preview :books:: https://cloudstack-documentation--348.org.readthedocs.build/en/348/

### CODE REVIEW

**Comment by shwstppr** [comment]
@blueorangutan docbuild

**Comment by DaanHoogland** [comment]
is this lgty @shwstppr ?

**Comment by rohityadavcloud** [comment]
@blueorangutan docbuild

**Review by shwstppr — APPROVED** [review]
LGTM, makes sense to remove

File: source/adminguide/networking/external_firewalls_and_load_balancers.rst [hunk 1]
```diff
@@ -17,10 +17,9 @@
 External Firewalls and Load Balancers
 -------------------------------------
 
-CloudStack is capable of replacing its Virtual Router with an external
-Juniper SRX device and an optional external NetScaler or F5 load
-balancer for gateway and load balancing services. In this case, the instances
-use the SRX as their gateway.
+CloudStack is capable of replacing its Virtual Router with an optional
```
Comment (andrijapanicsb) [inline source/adminguide/networking/external_firewalls_and_load_balancers.rst:20]:
```suggestion
CloudStack is capable of replacing some of its Virtual Router networking functionality by moving/offloading them
```

File: source/adminguide/networking/external_firewalls_and_load_balancers.rst [hunk 2]
```diff
@@ -17,10 +17,9 @@
...
+external NetScaler for gateway and load balancing services.
```
Comment (andrijapanicsb) [inline source/adminguide/networking/external_firewalls_and_load_balancers.rst:21]:
```suggestion
to some of the external network service providers such as e.g. NetScaler.
```

**Comment by andrijapanicsb** [comment]
LGTM, I reworded it a bit, but good to merge.

**Pull request merged**